data/data/agent/systemd/units/agent-register-infraenv.service

```diff
@@ -0,0 +1,27 @@
+[Unit]
+Description=Service that registers the infraenv
+Wants=network-online.target assisted-service.service
+PartOf=assisted-service-pod.service
+After=network-online.target assisted-service.service agent-register-cluster.service
+ConditionPathExists=/etc/assisted/node0
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+Environment=OPENSHIFT_INSTALL_RELEASE_IMAGE_MIRROR={{.ReleaseImageMirror}}
+EnvironmentFile=/etc/assisted/rendezvous-host.env
+EnvironmentFile=/usr/local/share/assisted-service/agent-images.env
+EnvironmentFile=/usr/local/share/assisted-service/assisted-service.env
+ExecStartPre=/bin/rm -f %t/%n.ctr-id
+ExecStartPre=/usr/local/bin/wait-for-assisted-service.sh
```

Comment:
Do you still need this since its done in `agent-register-cluster.service` which the service runs after?

Reply:
Good point. I will remove the wait-for-assisted-service.sh.